src/tests/jsonwalker.py

```diff
@@ -93,17 +89,15 @@ def _walk(self, adict):
         with open(options.filename, 'r') as f:
             content = list()
             for l in f:
-                content.append(cjson.decode(l.rstrip()))
+                content.append(json.loads(l.rstrip()))
         f.close()
     else:
-        print('Input file in jason format is required')
+        print('Input file in json format is required')
         exit()
 
     defaults = None
     if options.defaults is not None:
-        with open(options.defaults, 'r') as f:
-            defaults = cjson.decode(f.read())
-        f.close()
+        defaults = json.load(open(options.defaults, 'r'))
```

Comment:
Why remove the ``with ...``?  Leaking the file is harmless but there's no reason to add a harmless leak in a fix like this.  (However, the ``f.close()`` was wrong; I'm not sure f is even in scope at that point.)

Reply:
Whoops, meant to make one change or the other and did both... will fix, thanks!